src/main/java/core/basesyntax/model/dto/FruitDataDto.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model.dto;
+
+public class FruitDataDto {
+    private String operationType;
+    private String fruitName;
+    private Integer fruitQuantity;
+
+    public FruitDataDto(String operationType, String fruitName, Integer fruitQuantity) {
+        this.operationType = operationType;
+        this.fruitName = fruitName;
+        this.fruitQuantity = fruitQuantity;
+    }
+
+    public String getOperationType() {
+        return operationType;
+    }
+
+    public String getFruitName() {
+        return fruitName;
+    }
+
+    public Integer getFruitQuantity() {
+        return fruitQuantity;
+    }
+
+    public enum Operations {
+        b, s, p, r;
```

Comment:
I am not sure about this naming, I heard that now we should use more informative names.

Reply:
agree

Reply:
agree

Reply:
changed